<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>566</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QWidget" name="widgetLeftBottom" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>190</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>190</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;时间基准&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;飞机架次：B-1234 任务编号：D1&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;飞机架次：B-1204 任务编号：D5&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widgetRightBottom" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>190</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>190</height>
         </size>
        </property>
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="horizontalSlider">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QPushButton" name="pushButton_2">
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="horizontalSlider_5">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QPushButton" name="pushButton_3">
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="horizontalSlider_6">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QPushButton" name="pushButton_4">
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="horizontalSlider_7">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="widgetLeftTop" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QTreeWidget" name="treeWidgetProjectDirectory">
           <column>
            <property name="text">
             <string>工程项目</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QTreeWidget" name="treeWidgetProjectConfiguration">
           <column>
            <property name="text">
             <string>工程配置</string>
            </property>
           </column>
           <item>
            <property name="text">
             <string>飞机架次     B-1234</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>工程路径     C:</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>读写权限     可读可写</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>依赖环境</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="CustomTreeWidget" name="treeWidgetTainData">
           <column>
            <property name="text">
             <string>训练数据</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="CustomTreeWidget" name="treeWidgetTrainEvaluation">
           <column>
            <property name="text">
             <string>训练评价</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QTreeWidget" name="treeWidgetDataDisplay">
           <column>
            <property name="text">
             <string>数据展示</string>
            </property>
           </column>
           <item>
            <property name="text">
             <string>飞机编号     B-1234</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>速度           900km/h</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>位置           40°N,116°E,1000m</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>飞机过载     50KG</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>迎角           32°</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>剩余燃油     100L</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>飞行时长     2h</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widgetRightTop" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QTabWidget" name="tabWidget">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="view1">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <attribute name="title">
             <string>视图1</string>
            </attribute>
           </widget>
           <widget class="QWidget" name="view2">
            <attribute name="title">
             <string>视图2</string>
            </attribute>
           </widget>
           <widget class="QWidget" name="view3">
            <attribute name="title">
             <string>视图3</string>
            </attribute>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>工程</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
    <addaction name="actionDelete"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionImport"/>
    <addaction name="actionExport"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_LecturesAndPerformanceManagement">
    <property name="title">
     <string>讲评与成绩管理</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuConfiguration">
    <property name="title">
     <string>配置</string>
    </property>
    <addaction name="actionWorkSpace"/>
    <addaction name="actionViewArrangement"/>
   </widget>
   <widget class="QMenu" name="menuSet">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="actionAppearance"/>
    <addaction name="actionSoftwareVersion"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionHelpDocument"/>
    <addaction name="actionBeginnerTutorial"/>
   </widget>
   <widget class="QMenu" name="menuAccount">
    <property name="title">
     <string>账户</string>
    </property>
    <addaction name="actionLogin"/>
    <addaction name="actionLogout"/>
   </widget>
   <addaction name="menuProject"/>
   <addaction name="menu_LecturesAndPerformanceManagement"/>
   <addaction name="menuConfiguration"/>
   <addaction name="menuSet"/>
   <addaction name="menuHelp"/>
   <addaction name="menuAccount"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action1"/>
  </widget>
  <action name="action1">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>新建</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>关闭</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>导入</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>导出</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionWorkSpace">
   <property name="text">
    <string>工作区</string>
   </property>
  </action>
  <action name="actionViewArrangement">
   <property name="text">
    <string>视图排列</string>
   </property>
  </action>
  <action name="actionAppearance">
   <property name="text">
    <string>外观</string>
   </property>
  </action>
  <action name="actionSoftwareVersion">
   <property name="text">
    <string>软件版本</string>
   </property>
  </action>
  <action name="actionHelpDocument">
   <property name="text">
    <string>帮助文档</string>
   </property>
  </action>
  <action name="actionBeginnerTutorial">
   <property name="text">
    <string>新手教程</string>
   </property>
  </action>
  <action name="actionLogin">
   <property name="text">
    <string>登录</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header location="global">customtreewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
